render/render.js

```diff
@@ -521,7 +521,7 @@ module.exports = function($window) {
 		if (key in element) element[key] = typeof value === "function" ? callback : null
 		else {
 			var eventName = key.slice(2)
-			if (vnode.events === undefined) vnode.events = {}
+			if (!vnode.events) vnode.events = {}
```

Comment:
I think this does not what you want to achieve. It's the initializer if no events are bound to the vnode at all.

I suppose the event=null case is already supported. 526 unbinds current event if exist. New even is only bound if typeof `function`. So nothing to do here